templates/article/article.js

```diff
@@ -12,6 +12,15 @@ function restructureArticle(container, targetClass) {
 function decorate(doc) {
   const main = doc.querySelector('main');
   restructureArticle(main, '.hero');
+  const tocFlag = document.querySelector('meta[name="toc"]');
+  if (tocFlag && tocFlag.content !== 'no' && tocFlag.content !== 'false') {
+    const tocSection = document.createElement('div');
```

Comment:
Same here: dom-builder.js. But might be a overkill for creating a div with a single CSS class